var/spack/repos/builtin/packages/cctools/package.py

```diff
@@ -56,6 +60,16 @@ def patch(self):
             for m in makefiles:
                 filter_file('-fstack-protector-all', '', m)
 
+        # Remove two troublesome test cases:
+
+        # TR_parrot_dir.sh fails b/c of variations in how Linux kernels
+        # deal with open() on directories.
+        os.unlink("parrot/test/TR_parrot_dir.sh")
+
+        # TR_parrot_execve.sh fails b/c of the complexities of moving
+        # a python interpreter around.
+        os.unlink("parrot/test/TR_parrot_execve.sh")
```

Comment:
Are these tools present in every version of the package?  If they may not exist in some version, then you'll want to check first.

Reply:
Yes, these are integral test scripts in the repo.  it turns out that they are rather platform sensitive, so I'm removing them here while we ponder how to fix them upstream..